acceptance/lib/helper.rb

```diff
@@ -34,7 +34,7 @@ def self.initialize_config(options)
     puppet_build_version = get_option_value(options[:puppet_build_version],
                          nil, "Puppet Agent Development Build Version",
                          "PUPPET_BUILD_VERSION",
-                         "7f71e72e936f460e136b25880c6792d9a20decdf",
+                         "8e8866b3da044f405a4e651cf374ab24a435767f",
```

Comment:
I blame it on kerminator. He lied to me. I certainly wasn't the one that misread this. :D